app/gather/api/views.py

```diff
@@ -16,11 +16,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
+from django.conf import settings
+from rest_framework import status
 from rest_framework.viewsets import ModelViewSet
+from rest_framework.response import Response
+from rest_framework.decorators import action
 
 from aether.sdk.multitenancy.views import MtViewSetMixin
 from .models import Survey, Mask
 from .serializers import SurveySerializer, MaskSerializer
+from .utils import (
+  configure_consumers,
+  delete_survey_subscription,
+)
+from aether.sdk.multitenancy.utils import get_current_realm
+from django.utils.translation import gettext as _
```

Comment:
Note for the future: it's more clear if the import statements have an order

1. python standard modules
2. 3rd party modules
3. django and rest-framework
4. our external modules (like ather.python and aether.sdk)
5. current app modules